stable/kubecost/values.yaml

```diff
@@ -57,9 +57,6 @@ cost-analyzer:
     enabled: true
     # storageClass: "-"
 
-  podSecurityPolicy:
-    enabled: false
```

Comment:
Assuming the default is `true1`?

Reply:
yes. True is the default.

Reply:
maybe not `true1`, but definitely `true` 😉